modules/ROOT/pages/database-administration/syntax.adoc

```diff
@@ -105,6 +105,7 @@ YIELD { * \| field[, ...] } [ORDER BY field[, ...]] [SKIP n] [LIMIT n]
 [source, syntax, role="noheader"]
 ----
 CREATE DATABASE name [IF NOT EXISTS]
+[DEFAULT LANGUAGE CYPHER {5\|25}]
```

Comment:
what about the `CREATE OR REPLACE DATABASE name`, `CREATE COMPOSITE DATABASE name [IF NOT EXISTS]` and `CREATE OR REPLACE COMPOSITE DATABASE name` cases below? shouldn't they also have it?